scripts/psnr-test.sh

```diff
@@ -16,11 +16,20 @@
 #
 
 SCRIPT_DIR=$(dirname $0)
-OUTPUT_FRAME=$1
-INPUT_FRAME_DIR=$2
-RESIZED_FRAME_DIR=$3
+OUTPUT_FRAME="$1"
+INPUT_FRAME_DIR="$2"
+RESIZED_FRAME_DIR="$3"
+QR_CODE_DECODER="zbarimg"
 
-FRAME_NUMBER=$(java -jar $SCRIPT_DIR/javase-3.2.2-SNAPSHOT-jar-with-dependencies.jar $OUTPUT_FRAME |head -3|tail -1)
+
+PATH_TO_QR_DECODER=$(which $QR_CODE_DECODER)
```

Comment:
would it make sense to leave the old decoder in a comment for future reference?

Reply:
i figured the git history would be enough (i did still leave the jar in the repo).  do you think i should add the old one back and leave it commented?

Reply:
Actually feel free to disregard my comment. I suppose you have tested with zbarimg and you got no false negatives thus I don't see why we would ever want to go back to ZXing.